My_blog ROR app creation procedure.txt

```diff
@@ -24,4 +24,54 @@ test:
 8. rails g migration create_likes author_id:integer post_id:integer
 9. foreign keys added manually. Can be added by rails command also.
 10. indexes added manually. Can be added by rails command also.
-11. rails db:migrate
\ No newline at end of file
+11. rails db:migrate
+12. models(comment.rb, like.rb, post.rb & user.rb) created. models follows namming convention and they must be singular
+  a: comment.rb
+    class Comment < ApplicationRecord
+      belongs_to :post, class_name: 'Post' #it says Comment belongs_to post & post's class is Post
+      belongs_to :author, class_name: 'User' #it says Comment belongs_to author & author's class is User
+
+      after_create :update_comments_counter #it instructs the Comment class to run update_comments_counter.
+
+      def update_comments_counter
+        post.update(comments_counter: post.comments.count) #it says update comments_counter for the specific post by applying count(built-in) method on comments on a specific post.
+      end
+    end
+  b:  like.rb  
+    class Like < ApplicationRecord
+      belongs_to :author, class_name: 'User' #it says Like belongs_to author & author's class is User.
+      belongs_to :post, class_name: 'Post' #it says Like belongs_to post & post's class is Post.
+
+      after_create :update_likes_counter #it instructs the Like class to run update_likes_counter on creating like.
```

Comment:
Kudos for documenting your procedure this way! 🚀 This is going to boost your learning a lot. Keep it up! 👏🏼